### PR TITLE
Deleting unnecessary stats jobs

### PR DESCRIPTION
**Pull request by Lanaparezanin**
Before migration, we decided to remove unnecessary stats jobs. Jobs were removed based on the list in this work item:

https://github.com/orgs/NuGet/projects/21/views/1?filterQuery=milestone%3A%22Sprint+2024-11%22+assignee%3A%40me&pane=issue&itemId=64259916&issue=NuGet%7CEngineering%7C5445

Jobs removed:
Stats.RollUpDownloadFacts
Stats.CollectAzureCdnLogs
Stats.CreateAzureCdnInWarehouseReports
Stats.ImportAzureCdnStatistics

Projects removed:
Stats.LogInterpretation
Stats.RefreshClientDimension

Modified to exclude file related to a removed job, project not removed:
Stats.Warehouse

### CODE REVIEW

File: src/NuGet.Services.Storage/AggregateStorage.cs [hunk 1]
```diff
@@ -111,9 +111,12 @@ public override async Task<IEnumerable<StorageListItem>> ListAsync(bool getMetad
             return await _primaryStorage.ListAsync(getMetadata, cancellationToken);
         }
 
+        public override async Task<IEnumerable<StorageListItem>> ListTopLevelAsync(bool getMetadata, CancellationToken cancellationToken) =>
```
Comment (agr) [inline src/NuGet.Services.Storage/AggregateStorage.cs:114]: These changes are not related to deleting jobs, right? Should they go to a separate PR?

**Comment by erdembayar** [comment]
@Lanaparezanin 
Please enumerate all the stats jobs you're removing part of this PR in description. It'll help DRI diagnosing the problem in case something is regressed.

File: src/NuGet.Services.Storage/AggregateStorage.cs (hunk 1 above)
Comment (Lanaparezanin) [reply src/NuGet.Services.Storage/AggregateStorage.cs:114]: That makes sense actually, the reason I put it in this PR is because I was testing everything together. I will look into how to split this now.

**Review by agr — APPROVED** [review]

**Comment by agr** [comment]
Description could still use more details: it currently lists 4 jobs, but the changes include deleting 7 projects. What are those 3 extra projects?

**Comment by erdembayar** [comment]
> Description could still use more details: it currently lists 4 jobs, but the changes include deleting 7 projects. What are those 3 extra projects?

@Lanaparezanin  
Could you please address this comment?

**Comment by Lanaparezanin** [comment]
> Description could still use more details: it currently lists 4 jobs, but the changes include deleting 7 projects. What are those 3 extra projects?

Thanks for the comment, updated!
